src/components/ids-accordion/ids-accordion-panel.js

```diff
@@ -348,11 +348,10 @@ export default class IdsAccordionPanel extends Base {
    * @returns {void}
    */
   select(panel) {
-    if (panel === undefined) {
-      return;
+    if (panel && panel.tagName === 'IDS-ACCORDION-PANEL') {
```

Comment:
```suggestion
    if (panel?.tagName === 'IDS-ACCORDION-PANEL') {
```

Can also do this i think?